ros/processor/garbage_collector.py

```diff
@@ -50,3 +52,10 @@ def remove_outdated_data(self):
                 "due to the following error %s.",
                 self.prefix, str(error)
             )
+
+
+if __name__ == "__main__":
+    start_http_server(int(METRICS_PORT))
+    commence_cw_log_streaming('ros-processor')
```

Comment:
I think for every processor we should have different logging stream? instead of `ros-processor` commonly used?

Reply:
Agree I will change the log stream.

Reply:
If we have different logging streams for engine and events, IMO it will be a bit more work to correlate if any _x_ system has been updated after creation, thoughts @upadhyeammit @patilsuraj767 ?